object/token.go

```diff
@@ -137,18 +159,39 @@ func GetTokenByRefreshToken(refreshToken string) (*Token, error) {
 	return &token, nil
 }
 
-func GetTokenByTokenValue(tokenValue, tokenTypeHint string) (*Token, error) {
+func GetTokenByRefreshTokenOrName(refreshToken, tokenName string) (*Token, error) {
```

Comment:
this function is totally  same with `GetTokenByAccessTokenOrName`, it should be remove and name as `GetTokenByTokenOrName`

Reply:
you can just pack the same code to a new func instead of remove them

Reply:
hi @dacongda , i've just updated

Reply:
> hi @dacongda , i've just updated

I know, and those code is same with previous one,  I just suggest that make the same part between `GetTokenByRefreshTokenOrName` and GetTokenByAccessTokenOrName to be a new function

Reply:
@dacongda, these two methods are distinct: one retrieves tokens by RefreshTokenHash, while the other retrieves them by AccessTokenHash

Reply:
```go

		owner, name := util.GetOwnerAndNameFromIdNoCheck(tokenId)
		token = Token{Owner: owner, Name: name}
		existed, err = ormer.Engine.Get(&token)
		if err != nil {
			return nil, err
		}
	
```

these line can be a new function named `getTokenByTokenId`

Reply:
hi @dacongda, i've updated